servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/PublisherFlatMapSingleTest.java

```diff
@@ -639,4 +644,23 @@ void testRequestAndEmitConcurrency() throws Exception {
         assertThat("Unexpected items emitted.", received, hasSize(totalToRequest));
         assertThat(received, containsInAnyOrder(range(0, totalToRequest).boxed().toArray()));
     }
+
+    @Test
+    void testConcurrentEmitions() throws ExecutionException, InterruptedException {
+        final int maxSingles = 1_000;
```

Comment:
Is it reproduciable with 10s or 100s?

Reply:
Yes, but the higher the number the more reliable it is.